info.csv

```diff
@@ -6,6 +6,6 @@ Nitesh Saxena, 18SCSE1070012, 18021070012, Web Development, 1, 7, NA, Backend, N
 Cheenu Soni, 17SCSE1012016, 1713119007, Android Development, 2, 1, Java(Android) coding, "Competitve Coding, Android-FullStack Development, Algos", None, cheenusoni10@gmail.com, https://github.com/NullByte08, https://www.linkedin.com/in/cheenu-s-70b811118/, None
 Shashank aggarwal,18gcebit104,1809713094,web development,1st,NA,"C,C++,Python",Web development,NA,shashankaggarwal13@gmail.com,https://github.com/Sshashank18,https://www.linkedin.com/in/shashank-a-27055b123/,NA
 Ajay pratap singh,17scse101198,1713101188,Android,2,14,java, java and python, NA,pratap9411@gamil.com,https://github.com/ajay9411,https://www.linkedin.com/in/ajay9411,NA
-ANJALI SINGH,17SCSE101722,1713101689,Android development,2,10,python c c++ java core,coding,NA,attri2707@gmail.com,https://github.com/anjiattri,https://www.linkedin.com/in/anjali-singh-947ab0179/
+ANJALI SINGH,17SCSE101722,1713101689,Android development,2,10,python c c++ java core,coding,NA,attri2707@gmail.com,https://github.com/anjiattri,https://www.linkedin.com/in/anjali-singh-947ab0179/,NA
 Mubashshira Farooqui, 17scse101157,1713101149,Web Development, 2,5, Java( Web sites n Android), "competitive coding, backend",payroll system, mubashshira.farooqui2000@gmail.com,https://github.com/Mubashshira,https://www.linkedin.com/in/mubashshira-farooqui-249aaa160/,NA
 
```

Comment:
Nikhil Goyal , 18SCSE1010187 , 18021011430 , Android , 1 , 8 ,C Coding , Developing Apps , Competitive Coding , nikhilgoyal145@gmail.com , https://github.com/nikhilgoyal145 , www.linkedin.com/in/nikhil-goyal-7307a317b , NA